db/migrate/20191126190129_add_itsm_modeling.rb

```diff
@@ -0,0 +1,20 @@
+class AddItsmModeling < ActiveRecord::Migration[5.2]
+  def change
+    create_table :order_templates do |t|
+      t.string :description
+      t.string :name
+      t.jsonb :post_hash
```

Comment:
What are the `jsonb` hash columns intended for?  Configuration settings?  If so maybe we do not need to add them at this time.